src/com/lftechnology/java/training/sanish/game/FootballLeague.java

```diff
@@ -0,0 +1,28 @@
+package com.lftechnology.java.training.sanish.game;
+
+import java.util.ArrayList;
+import java.util.logging.Logger;
+
+/**
+ * Football League game result display program
+ *
+ * @author Sanish Maharjan <sanishmaharjan@lftechnology.com>
+ */
+public class FootballLeague {
+    private final static Logger LOGGER = Logger.getLogger(FootballLeague.class.getName());
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Reorder the modifiers to comply with the Java Language Specification. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AModifiersOrderCheck)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "LOGGER" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)